language-plutus-core/test/Normalization/Type.hs

```diff
@@ -16,6 +16,22 @@ import           Test.Tasty
 import           Test.Tasty.Hedgehog
 import           Test.Tasty.HUnit
 
+-- test [rec (lam dat (fun (type) (type)) [dat a])]
+test_catamorphism :: IO ()
```

Comment:
Why is it called `catamorphism`? You really want to test that a type application is neutral whenever its head is neutral and the argument is normal?